migcontrol/templates/base.html

```diff
@@ -137,14 +145,14 @@ <h3>{% trans "Subscribe to our monthly review" %}</h3>
         <a href="">{% trans "Show all issues" %} &gt;</a>
       </p>
       <div class="row">
-        <form class="col-md-5 offset-md-4">
-          <div class="row g-2">
+        <form class="col-md-5 offset-md-4" id="form">
```

Comment:
I would suggest that you refer to the grid layout again here:

https://getbootstrap.com/docs/5.1/layout/grid/

It's based on the number "12". So we divide each row into 12 columns. In this case, when we have a form that has a column of 5 and an offset of 4, that takes up "9" columns out of 12. There will be 3 left. So you have 4 on one side and 3 on the other. So it isn't centered. To make it centered, you could for instance have the form be 6 columns with an offset of 3 or 4 columns with an offset of 4.